SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -444,6 +449,15 @@ TArray<UClass*> GetAllSupportedClasses()
 			continue;
 		}
 
+		// Avoid processing classes contained in Directories to Never Cook
+		bool bNeverCook = false;
+		if (FDirectoryPath* ContainingDirectory = DirectoriesToNeverCook.FindByPredicate([SupportedClass](FDirectoryPath Directory) {
```

Comment:
nit: No reason to create a variable for this, just see if FindByPredicate doesn't return a nullptr

Reply:
Done